kclvm/tools/src/format/test_data/format_data/union_types.golden

```diff
@@ -0,0 +1,5 @@
+workloadType: "Deployment" | "StatefulSet" = "Deployment
```

Comment:
s/"Deployment/"Deployment"

Reply:
My mistake, fixed. (it seems the copilot sometimes suggests completion with unclosed parentheses and quotation marks.  :D

Reply:
> My mistake, fixed. (it seems the copilot sometimes suggests completion with unclosed parentheses and quotation marks. :D

:) 😂